docs/Upgrade.md

```diff
@@ -0,0 +1,48 @@
+# Upgrade Guide
+
+This guide is used to upgrade Harbor deployed by chart since version 0.3.0.  
+
+**Notes**:
+- As the database scheme may change between different versions of Harbor, there is a progress to migrate the scheme during the upgrade and the downtime cannot be avoid  
+- The database scheme cannot be downgraded automatically, so the `helm rollback` is not supported  
+
+## Upgrade
+1. **Backup database**  
+Backup the database used by Harbor in case the upgrade process fails.  
+2. **Download new chart**  
+Download the latest version of Harbor chart.  
+3. **Configure new chart**  
+Configure the new chart to make sure that the configuration items have the same values with the old one.  
```

Comment:
This sounds like a lot work, shall we tell user if there's no big change in `values.yaml` he just make necessary change to the previous one.  But for 0.3.0 to the latest one, he needs to manually rewrite it.

Reply:
Will do that in the doc at next version as the manual process is needed in the release.